relay-quotas/Cargo.toml

```diff
@@ -21,6 +21,7 @@ redis = [
 [dependencies]
 relay-common = { path = "../relay-common" }
 relay-log = { path = "../relay-log", optional = true }
+relay-dynamic-config = { path = "../relay-dynamic-config" }
```

Comment:
This pulls some heavy dependencies into the previously lightweight `relay-quotas`. But the number of indirect dependencies will be reduced once we remove them from `relay-dynamic-config`.

As an alternative intermediate solution, we could add `relay-quotas` as a dependency to `relay-dynamic-config`, and move the `Quota` type back to where it was.

Reply:
I highly favor the alternative solution you propose: Pull `relay-quotas` as a dependency into `relay-dynamic-config`. That's the true direction: The dynamic config includes quota declarations. For this reason, `relay-quota` comes with feature flags that disable all of the quota enforcement.

We should strive for smaller semantic crates that each ship their own data and configuration types, and then higher-level crates that combine these. 

If you feel like `relay-dynamic-config` is getting too large, introduce a trait: 
- Make the logic around dealing with configs dynamic
- Declare the large, composite config type at a place where you have access to everything (e.g. back in the server).